--- Makefile	Sun Jul  7 22:09:25 2002
+++ Makefile.new	Mon May 26 14:33:05 2003
@@ -27,8 +27,7 @@
 	@tools/autoconf
 	@echo [*] Creating Menu, please wait.
 	@echo This needs the 'ncurses' library. If it is not available, menuconf wont work. If you are using 'curses', use make menuconfig-curses instead.
-	@$(CC) menuconf/menuconf.c menuconf/inputbox.c menuconf/util.c menuconf/checklist.c menuconf/menubox.c menuconf/textbox.c src/snprintf.c -I. -lncurses -lm -o menuconf/menuconf 2>/dev/null
-	@menuconf/menuconf
+	@$(CC) menuconf/menuconf.c menuconf/inputbox.c menuconf/util.c menuconf/checklist.c menuconf/menubox.c menuconf/textbox.c src/snprintf.c -I. -I/usr/include/ncurses -lncurses -lm -o menuconf/menuconf 2>/dev/null
 	@clear
 	@echo Now compile psyBNC using make, if not yet compiled, or if Options were changed.
 	@echo done.
